### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by danintel**
- Add introduction to Avalon video by Manoj
- Add introduction to Avalon video by Eugene
- Add answer about proxy/direct mode
- Move introductory videos to beginning of list
- Move error message question to correct section

Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

File: docs/FAQ.rst [hunk 1]
```diff
@@ -42,6 +42,16 @@ The TCS is defined by the Enterprise Ethereum Alliance (EEA) Task Force and
 is at
 https://entethalliance.github.io/trusted-computing/spec.html
 
+What is the difference between Proxy Mode and Direct Mode?
+----------------------------------------------------------
+Proxy Mode is where the requester (client) executes a smart contract
+(or chaincode or similar) on the blockchain.
+Th smart contract invoktes an Avalon executes an
```
Comment (manojgop) [inline docs/FAQ.rst:49]: Th -> The
invoktes an -> invokes
The smart contact event invokes Avalon and Avalon executes

File: docs/FAQ.rst [hunk 2]
```diff
@@ -42,6 +42,16 @@ The TCS is defined by the Enterprise Ethereum Alliance (EEA) Task Force and
...
+Avanlon workload processor through the Blockchain Connector
```
Comment (manojgop) [inline docs/FAQ.rst:50]: Avanlon  -> Avalon

File: docs/FAQ.rst [hunk 3]
```diff
@@ -42,6 +42,16 @@ The TCS is defined by the Enterprise Ethereum Alliance (EEA) Task Force and
...
+interface for that blockchain platform (Ethereum, Fabric, etc.).
+Direct mode is where the requester (client) operates directly with
+Avalon workload processors, without a blockchain.
```
Comment (manojgop) [inline docs/FAQ.rst:53]: Avalon workload processors -> Avalon (via HTTP-JRPC listener)

Comment (danintel) [reply docs/FAQ.rst:53]: Fixed

File: docs/FAQ.rst (hunk 2 above)
Comment (danintel) [reply docs/FAQ.rst:50]: Fixed.

File: docs/FAQ.rst (hunk 1 above)
Comment (danintel) [reply docs/FAQ.rst:49]: Fixed.

**Pull request merged**